openapi/templates/setup.mustache

```diff
@@ -31,7 +31,7 @@ class Install(InstallCommand):
     """ Customized setuptools install command which uses pip. """
 
     def run(self, *args, **kwargs):
-        subprocess.call([sys.executable, '-m', 'pip', 'install','git+https://github.com/onepanelio/couler@dev'])
+        subprocess.call([sys.executable, '-m', 'pip', 'install','git+https://github.com/onepanelio/couler@v0.1.1rc8'])
```

Comment:
If https://github.com/couler-proj/couler/pull/176 is merged before we cut this release, then we can update this to point to upstream.